### DISCUSSION TITLE
Does POP work in Mail?

### DISCUSSION THREAD

**Discussion by binaryboyo** [elementary/mail#976] [Q&A]
Hi All,

Mail seems to only want to add an account from System Settings and the only option shown is IMAP !?

When this option is selected and configured with POP working info.  it goes away for a while and comes back with ´Could not save the email A/C. TLS connection closed´?

Does Mail support POP and if so how? And if not, what email client does? 

New to Elementary/Linux so unsure what to do but need to migrate email  A/C  across.

Any ideas, suggestions or solutions would be greatly appreciated!

Cheers,

BB

**Comment by binaryboyo** [comment]
Bump! :/

**Comment by lenemter** [comment]
Which email provider are you trying to add?

**Reply by binaryboyo** [reply]
Hotmail, which works OK on other devices.

**Reply by lenemter** [reply]
Not sure if live and hotmail emails are the same, but you might have encountered the issue described in https://github.com/elementary/mail/issues/964, which has a workaround.

**Reply by binaryboyo** [reply]
Thanks however that post seems to turn POP off!

I need to use POP protocol as I always have.

So can POP be used on Mail and if so how? Or do I need another mail client and if so what?
